test/integration/other/analytics-queries.js

```diff
@@ -101,7 +101,11 @@ const submitToForm = (service, user, projectId, xmlFormId, xml, deviceId = 'abcd
 ////////////////////////////////////////////////////////////////////////////////
 // Tests!
 ////////////////////////////////////////////////////////////////////////////////
-describe('analytics task queries', () => {
+// eslint-disable-next-line func-names
+describe('analytics task queries', function () {
+  // increasing timeouts on this set of tests
+  this.timeout(8000);
+
```

Comment:
This change should be introduced separately.  Separate PR at https://github.com/getodk/central-backend/pull/967.

Reply:
#967 has been merged, so this change can be reverted from this PR.